apps/web/src/index.css

```
@@ -172,30 +175,23 @@
 
 @utility btn-primary {
-  /* Button Components */
-  @apply bg-primary text-primary-foreground hover:bg-primary/90 px-6 py-2 rounded-md font-medium transition-colors;
-
-  &:hover {
-    @apply bg-primary/90;
-  }
+  @apply bg-gradient-button hover:bg-gradient-button-hover text-white font-medium px-6 py-2 rounded-md transition-all duration-200;
 }
 
 @utility btn-primary-large {
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@utility" (at-rule-no-unknown)

(at-rule-no-unknown)